connectors/aws/configurations/config.json

```diff
@@ -3,6 +3,9 @@
         "type": {
             "type": "connectorType",
             "displayName": "AWS"
+        },
+        "region": {
```

Comment:
I don't think you need to add region if it's not used by the CAR connector. You just need to rename the properties that already exist.